tests/Rules/ForbiddenThisArgumentRule/ForbiddenThisArgumentRuleTest.php

```diff
@@ -23,16 +23,16 @@ public function testRule(string $filePath, array $expectedErrorMessagesWithLines
 
     public static function provideData(): Iterator
     {
-        yield [__DIR__ . '/Fixture/SkipMethodExists.php', []];
-        yield [__DIR__ . '/Fixture/SkipReflectionCalling.php', []];
-        yield [__DIR__ . '/Fixture/SkipNotVariable.php', []];
-        yield [__DIR__ . '/Fixture/SkipNotThis.php', []];
-        yield [__DIR__ . '/Fixture/SkipExtendsKernel.php', []];
-
-        yield [__DIR__ . '/Fixture/StaticCall.php', [[ForbiddenThisArgumentRule::ERROR_MESSAGE, 13]]];
-        yield [__DIR__ . '/Fixture/ThisArgument.php', [[ForbiddenThisArgumentRule::ERROR_MESSAGE, 11]]];
-        yield [__DIR__ . '/Fixture/ThisArgumentCopy.php', [[ForbiddenThisArgumentRule::ERROR_MESSAGE, 12]]];
-        yield [__DIR__ . '/Fixture/ArgInFuncCall.php', [[ForbiddenThisArgumentRule::ERROR_MESSAGE, 11]]];
+        yield 'SkipNativeFunctionCalls' => [__DIR__ . '/Fixture/SkipNativeFunctionCalls.php', []];
```

Comment:
I think this disables the `--filter test#10` feature. Could you revert the keys?

Reply:
Sure, for me it was hard to see from the phpunit fail message exactly which of the cases failed, so i thought this would be a improvement to make the fails easier to understand, but when it breaks another feature i'll revert this change

Reply:
Yeah, please do.

This should be fixed in a PHPUnit, so both are supported at the same time.